fs/vfs/fs_symlink.c

```diff
@@ -180,4 +180,30 @@ int symlink(FAR const char *path1, FAR const char *path2)
   return ERROR;
 }
 
+/****************************************************************************
+ * Name: link
+ *
+ * Description:
+ *  The link function provides a wrapper to symlink. Solely to provide
+ *  compatibility to posix compatibility layer, in
+ *  libxx/libcxx/src/filesystem/posix_compat.h.
+ *
+ *  See symlink for details on limitations.
+ *
+ * Input Parameters:
+ *   path1 - Points to a pathname naming an existing file.
+ *   path2 - Points to a pathname naming the new directory entry to be
+ *           created.
+ *
+ * Returned Value:
+ *   On success, zero (OK) is returned.  Otherwise, -1 (ERROR) is returned
+ *   the errno variable is set appropriately.
+ *
+ ****************************************************************************/
+
+int link(FAR const char *path1, FAR const char *path2)
```

Comment:
is `link` required to be a function? Can we use macro instead?

Reply:
no, since the c++ header file normally contain somethin like this:
```
namespace std
{
  using ::link;
}
```